ethpm/assets/package.schema.v2.json

```diff
@@ -2,14 +2,14 @@
   "title": "Release Package Specification",
   "type": "object",
   "required": [
-    "spec_version",
+    "manifest_version",
```

Comment:
Assuming this file is largely copy/pasted from the `ethpm-spec` repository, maybe we should include that repository as a submodule to this one and just reference the spec through the submodule.  Should be way easier to stay up-to-date with the main spec that way.

Reply:
@pipermerriam not sure how long it'll take the schema in `ethpm-spec` to get updated? is it cool to go ahead and merge this (there's also some unrelated cleanup) - and i'll add referencing `ethpm-spec`'s spec via submodule as an issue here to be done once `ethpm-spec` pr's get merged?

Reply:
Go for it.